roles/ceph-update/tasks/main.yml

```diff
@@ -46,3 +64,12 @@
   command: ceph osd unset nodeep-scrub
   delegate_to: "{{ groups['ceph_monitors'][0] }}"
   when: inventory_hostname == groups['ceph_osds'][-1]
+
+- name: restart cinder services
+  service: name="{{ item }}" state=restarted
+  with_items:
+    - cinder-api
+    - cinder-scheduler
+    - cinder-volume
+    - cinder-backup
+  when: ceph.upgrade2jewel|bool and 'controller' in group_names
```

Comment:
yes, it's to load new ceph library.
ceph upgrade will not change cinder configuration, which can trigger cinder services restarted.
So we restart cinder services here